gc/base/Configuration.hpp

```diff
@@ -56,6 +56,9 @@ class MM_Configuration : public MM_BaseVirtual
 {
 /* Data members / types */
 public:
+	bool packetListSplitForced;
+	bool cacheListSplitForced;
+	bool splitFreeListAmountForced;
```

Comment:
class member variable names should start with `_`

Reply:
Fixed!